rfcs/text/0013_saved_object_migrations.md

```diff
@@ -248,45 +248,49 @@ Note:
+10. Update the mappings of the target index
    1. Retrieve the existing mappings including the `migrationMappingPropertyHashes` metadata.
-   2. Update the mappings with `PUT /.kibana_7.10.0_001/_mapping`. The API deeply merges any updates so this won't remove the mappings of any plugins that were enabled in a previous version but are now disabled.
+   2. Update the mappings with `PUT /.kibana_7.10.0_001/_mapping`. The API deeply merges any updates so this won't remove the mappings of any plugins that are disabled on this instance but have been enabled on another instance that also migrated this index.
    3. Ensure that fields are correctly indexed using the target index's latest mappings `POST /.kibana_7.10.0_001/_update_by_query?conflicts=proceed`. In the future we could optimize this query by only targeting documents:
       1. That belong to a known saved object type.
-      2. Which don't have outdated migrationVersion numbers since these will be transformed anyway.
-      3. That belong to a type whose mappings were changed by comparing the `migrationMappingPropertyHashes`. (Metadata, unlike the mappings isn't commutative, so there is a small chance that the metadata hashes do not accurately reflect the latest mappings, however, this will just result in an less efficient query).
-8. Transform documents by reading batches of outdated documents from the target index then transforming and updating them with optimistic concurrency control. 
-   1. Ignore any version conflict errors.
-   2. If a document transform throws an exception, add the document to a failure list and continue trying to transform all other documents. If any failures occured, log the complete list of documents that failed to transform. Fail the migration.
-9.  Mark the migration as complete. This is done as a single atomic
+11.  Mark the migration as complete. This is done as a single atomic
     operation (requires https://github.com/elastic/elasticsearch/pull/58100)
-    to guarantees when multiple versions of Kibana are performing the
+    to guarantee that when multiple versions of Kibana are performing the
     migration in parallel, only one version will win. E.g. if 7.11 and 7.12
     are started in parallel and migrate from a 7.9 index, either 7.11 or 7.12
     should succeed and accept writes, but not both.
-   3. Checks that `.kibana` alias is still pointing to the source index
-   4. Points the `.kibana_7.10.0` and `.kibana` aliases to the target index.
-   5. If this fails with a "required alias [.kibana] does not exist" error fetch `.kibana` again:
+   1. Checks that `.kibana` alias is still pointing to the source index
+   2. Points the `.kibana_7.10.0` and `.kibana` aliases to the target index.
+   3. Removes the temporary index `.kibana_7.10.0_reindex_temp`
```

Comment:
nit: changing the grammar here to actions rather than descriptions to be consistent with the rest of the RFC
```suggestion
   1. Check that `.kibana` alias is still pointing to the source index
   2. Point the `.kibana_7.10.0` and `.kibana` aliases to the target index.
   3. Remove the temporary index `.kibana_7.10.0_reindex_temp`
```